lib/ansible/modules/storage/netapp/netapp_e_lun_mapping.py

```diff
@@ -37,6 +39,18 @@
     description:
       - The name of the volume you wish to include in the mapping.
     required: True
+    aliases:
+      - volume
+  lun:
+    description:
+      - This option is deprecated and will have no effect on the module's outcome
+      - LUN number will be chosen by the storage array.
+    required: no
+  target_type:
+    description:
+      - This option is deprecated and will have no effect on the module's outcome
```

Comment:
Why has this been added if it's deprecated?

Reply:
@gundalow There was a patch submitted to resolve a different issue a few days ago. (#44666). This patch removed a few parameters from the module that we realized really weren't necessary. Unfortunately, removing those fields would cause existing playbooks utilizing those parameters to break (that's my fault for not remembering that). This patch is adding those back in (as deprecated fields), to maintain existing playbooks. Does that make sense?

Reply:
If these options don't do anything then I think it may be confusing for the end user to leave them in but not used